CONTRIBUTING.md

```diff
@@ -0,0 +1,188 @@
+# Contributing to ief <!-- omit from toc -->
```

Comment:
Maybe a code of conduct, similar to Mozilla's: https://www.mozilla.org/en-US/about/governance/policies/participation/ ?